content/blog/devtools_ui.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Chrome DevTools UI"
+date: 2021-01-11T09:00:00+02:00
+description: "Chrome DevTools has always been an essential feature in Sauce Labs for live testing real devices. This blog post reports about our recent revamp of the UI code."
+author: "Sakhi Mansoor"
+images:
+    - /images/blog/devtools-ui/after.jpg
+tags:
+    - ux
+    - frontend
+    - chrome
+    - devtools
+    - devtools-ui
+---
+
+Chrome DevTools has always been an essential feature in Sauce Labs for live testing real devices. This will allow a user to inspect web pages in Android devices during live testing and use Chrome DevTools to debug issues.
```

Comment:
I think this makes more sense to have `this allows a user` instead of `this will allow a user`. What do you think?

Reply:
updated to:

> Chrome DevTools has always been an essential feature in Sauce Labs for live testing real devices. It allows users to inspect web pages in Android devices and use Chrome DevTools to debug issues.